jetbrains-ultimate/src-201/software/aws/toolkits/jetbrains/services/rds/auth/IamAuthWidget.kt

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.rds.auth
+
+import com.intellij.database.dataSource.DataSourceUiUtil
+import com.intellij.database.dataSource.LocalDataSource
+import com.intellij.database.dataSource.url.template.ParametersHolder
+import com.intellij.database.dataSource.url.template.UrlEditorModel
+import com.intellij.database.dataSource.url.ui.UrlPropertiesPanel
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.components.JBTextField
+import org.jetbrains.annotations.TestOnly
+import software.aws.toolkits.jetbrains.services.rds.RdsResources
+import software.aws.toolkits.jetbrains.ui.AwsAuthWidget
+import software.aws.toolkits.resources.message
+import javax.swing.JPanel
+
+const val INSTANCE_ID_PROPERTY = "AWS.RdsInstanceId"
+
+class RdsAwsAuthWidget : AwsAuthWidget() {
```

Comment:
Make class name match file name since only one class

Reply:
Ah refactor didn't work properly because there's that floating const val, fixed